cmd/limactl/start.go

```diff
@@ -61,6 +61,7 @@ $ cat template.yaml | limactl start --name=local -
 	startCommand.Flags().Bool("tty", isatty.IsTerminal(os.Stdout.Fd()), "enable TUI interactions such as opening an editor, defaults to true when stdout is a terminal")
 	startCommand.Flags().String("name", "", "override the instance name")
 	startCommand.Flags().String("set", "", "modify the template inplace, using yq syntax")
+	startCommand.Flags().Bool("progress", false, "show progress")
```

Comment:
This is confusing, because the progress shown in the default mode

Reply:
Needs to be split